dll/win32/uxtheme/buffer.c

```diff
@@ -41,35 +64,104 @@ HRESULT WINAPI BufferedPaintUnInit(VOID)
 /***********************************************************************
  *      BeginBufferedPaint                                 (UXTHEME.@)
  */
-HPAINTBUFFER WINAPI BeginBufferedPaint(HDC hdcTarget,
-                                       const RECT * prcTarget,
-                                       BP_BUFFERFORMAT dwFormat,
-                                       BP_PAINTPARAMS *pPaintParams,
-                                       HDC *phdc)
+HPAINTBUFFER WINAPI BeginBufferedPaint(HDC targetdc, const RECT *rect,
+        BP_BUFFERFORMAT format, BP_PAINTPARAMS *params, HDC *retdc)
```

Comment:
F*ck wine

Reply:
I get that you're not a fan of the way WINE does things in some cases...but this PR is a for ReactOS, and I *already* have no idea what I'm doing, even without the added confusion.

Were you trying to help me or help get this PR wrapped up and merged by sharing your distaste for WINE in a comment here? If so, then...how?

Were you implying a recommended course of action on my part?